packages/edit-post/src/components/secondary-sidebar/list-view-outline.js

```diff
@@ -57,30 +57,84 @@ function EmptyOutlineIllustration() {
 }
 
 export default function ListViewOutline() {
-	const { headingCount } = useSelect( ( select ) => {
-		const { getGlobalBlockCount } = select( blockEditorStore );
-		return {
-			headingCount: getGlobalBlockCount( 'core/heading' ),
-		};
-	}, [] );
+	const { headingCount, paragraphCount, blockCount } = useSelect(
+		( select ) => {
+			const { getGlobalBlockCount, getBlocks } =
+				select( blockEditorStore );
+			const blocks = getBlocks();
+			return {
+				headingCount: getGlobalBlockCount( 'core/heading' ),
+				paragraphCount: getGlobalBlockCount( 'core/paragraph' ),
+				blockCount: blocks.length ? blocks.length : 0,
```

Comment:
The `TableOfContentsPanel` panel uses `getGlobalBlockCount` here as well.

https://github.com/WordPress/gutenberg/blob/074443a82f6a83e1800a2ed8c28e296165916967/packages/editor/src/components/table-of-contents/panel.js#L17-L27